adam-core/src/main/scala/org/bdgenomics/adam/models/ReferenceRegion.scala

```diff
@@ -144,10 +189,10 @@ object ReferenceRegion {
  *            which is <i>not</i> in the region -- i.e. [start, end) define a 0-based
  *            half-open interval.
  */
-case class ReferenceRegion(referenceName: String, start: Long, end: Long) extends Ordered[ReferenceRegion] with Interval {
+case class ReferenceRegion(referenceName: String, start: Long, end: Long) extends Interval {
```

Comment:
I don't know if there's anything stopping us, per se. I've just developed these feelings over the past few weeks (been playing around with a lot of stuff in avocado). The syntactical stuff can be fixed by having a `ReferenceRegion` object, I think, which we'd want anyways to implement the current functionality that we've got baked into `ReferenceRegion`.